packages/installCatalog.sh

```diff
@@ -8,11 +8,11 @@
 SCRIPTDIR="$(cd $(dirname "$0")/ && pwd)"
 OPENWHISK_HOME=${OPENWHISK_HOME:-$SCRIPTDIR/../../openwhisk}
 
-source "$SCRIPTDIR/validateParameter.sh" $1 $2 $3
-: ${WHISK_SYSTEM_AUTH:?"WHISK_SYSTEM_AUTH is not configured. Please input the correctly parameter: CATALOG_AUTH_KEY"}
-: ${WHISK_API_HOST:?"WHISK_API_HOST is not configured. Please input the correctly parameter: API_HOST"}
-: ${WHISK_CLI_PATH:?"WHISK_CLI_PATH is not configured. Please input the correctly parameter: cli_path"}
-
+source "$SCRIPTDIR/validateParameter.sh" $1 $2 $3 $4 $5
+: ${WHISK_SYSTEM_AUTH:?"WHISK_SYSTEM_AUTH is not configured. Please input the correct parameter: CATALOG_AUTH_KEY"}
+: ${WHISK_EDGE_HOST:?"WHISK_EDGE_HOST is not configured. Please input the correct parameter: API_HOST"}
```

Comment:
i think you want to use the api host not the edge host.

Reply:
oh i see, a bit confused on the namings. This is a duplicate for https://github.com/apache/incubator-openwhisk-catalog/pull/230 and I was reading the comments there. Trying to debug some failures on the existing PR. 

Looks like in that one WHISK_API_HOST was replaced by WHISK_EDGE_HOST (naming wise) Should this not have been the case?